internal/db/schema/migrations/oss/postgres/48/01_vault_credentials.up.sql

```diff
@@ -0,0 +1,210 @@
+
+  create view credential_vault_list_lookup_store as
+  select store.public_id                   as public_id,
+         store.project_id                  as project_id,
+         store.name                        as name,
+         store.description                 as description,
+         store.create_time                 as create_time,
+         store.update_time                 as update_time,
+         store.delete_time                 as delete_time,
+         store.version                     as version,
+         store.vault_address               as vault_address,
+         store.namespace                   as namespace,
+         store.ca_cert                     as ca_cert,
+         store.tls_server_name             as tls_server_name,
+         store.tls_skip_verify             as tls_skip_verify,
+         store.worker_filter               as worker_filter,
+         token.token_hmac                  as token_hmac,
+         coalesce(token.status, 'expired') as token_status,
+         cert.certificate                  as client_cert,
+         cert.certificate_key_hmac         as client_cert_key_hmac
+    from credential_vault_store store
+    left join credential_vault_token token
+      on store.public_id = token.store_id
+     and token.status = 'current'
+    left join credential_vault_client_certificate cert
+      on store.public_id = cert.store_id
+   where store.delete_time is null;
+
+  comment on view credential_vault_list_lookup_store is
+    'credential_vault_list_lookup_store is a view where each row contains a credential store. '
+    'If the Vault token has expired this view will return an empty token_hmac and a token_status of ''expired'' '
+    'No encrypted data is returned. This view can be used to retrieve data which will be returned external to boundary.';
+
+  create view credential_vault_client_store as
+  select store.public_id                   as public_id,
+         store.project_id                  as project_id,
+         store.vault_address               as vault_address,
+         store.namespace                   as namespace,
+         store.ca_cert                     as ca_cert,
+         store.tls_server_name             as tls_server_name,
+         store.tls_skip_verify             as tls_skip_verify,
+         store.worker_filter               as worker_filter,
+         token.token                       as ct_token, -- encrypted
+         token.token_hmac                  as token_hmac,
+         coalesce(token.status, 'expired') as token_status,
+         token.key_id                      as token_key_id,
+         cert.certificate                  as client_cert,
+         cert.certificate_key              as ct_client_key, -- encrypted
+         cert.key_id                       as client_key_id
+    from credential_vault_store store
+    left join credential_vault_token token
```

Comment:
Why does this view need to return a value when there is no `current` vault token?

Reply:
This view is used in the update path of the store, so it needs to return the store to update even if the token is expired.
It is also reused in the `credential_vault_library_issue_credentials` view, when issuing credentials we return an error if the any of the requested libraries have an expired token. We could modify this to do a len compare and validate the number of requests and returned libraries are equal, but the expired error is probably more clear.